README.md

```diff
@@ -11,7 +11,7 @@ You can also use forever from inside your own node.js code.
   var child = new (forever.Monitor)('your-filename.js', {
     max: 3,
     silent: true,
-    args: []
+    args: ['--color']
```

Comment:
To help others figure out how to enable color support. See [here](https://github.com/foreverjs/forever-monitor/issues/104) for details